### ISSUE TITLE
Confused about "No violations found in aXe results object" error

### ISSUE THREAD

**Issue by jfhector** [nickcolley/jest-axe#86]
First, thanks for this break tool.
It was very easy to set up.

#### The error I get

I get this error:

```
  ● <KFormInputLabel> › Does not have basic accessibility issues

    No violations found in aXe results object

      64 | 
      65 |     const a11yTestResults = configuredAxe(component.container);
    > 66 |     expect(a11yTestResults).toHaveNoViolations();
         |                             ^
      67 |   });
```

Here's my code:

```tsx
import * as React from "react";
import { render, cleanup } from "@testing-library/react";
import { toHaveNoViolations, axe } from "jest-axe";
import "@testing-library/jest-dom";

expect.extend(toHaveNoViolations);

/* more code */

describe("<KFormInputLabel>", () => {
  afterEach(() => {
    jest.resetAllMocks();
    cleanup();
  });

  /* more code */

  it("Does not have basic accessibility issues", async () => {
    const component = render(
      <KFormInputLabel htmlFor="dow3uef98i6">
        Label text
      </KFormInputLabel>
    );

    const a11yTestResults = axe(
      component.container
    );
    expect(a11yTestResults).toHaveNoViolations();
  });
});
```

#### I'm not sure what the error means, or what to do about it

"No violations found in aXe results object" sounds like the test should pass. So why does it fail?

Also, I can't see why I'm getting this error when testing this component. It didn't happen it other places where I've used the same test code.

Any thought or guidance would help.

**Comment by jfhector** [comment]
Ah, I had forgotten the `await` keyboard before calling the `axe` function.

**Comment by jfhector** [comment]
It might be useful to extend the error message to suggest that forgetting `await` might be the cause.

**Comment by adunkman** [comment]
> Ah, I had forgotten the `await` keyboard before calling the `axe` function.

Me too, thanks for posting this issue. 🙂
